app/main.py

```diff
@@ -2,0 +1,19 @@
+from app.errors import NotWearingMaskError, VaccineError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
Code Style: It's a good practice to use type annotations for all function parameters and the return type. For the `cafe` parameter, you could use the `Cafe` class as the type.